templates/admin/machine/index.html.ep

```diff
@@ -1,18 +1,26 @@
 % layout 'admin';
 % title 'Machines';
 
+% content_for 'ready_function' => begin
+$('#machines').DataTable( {
+"paging" : false,
+"lengthChange": false,
+"order": [],
+} );
+% end
+
```

Comment:
well, I tried but I have no templates in my database, so I couldn't see the problem.